feat(get-token): update coin image, tilt animation, and font styles

skajosborn opened the pull request:
### Summary

This PR improves the design and responsiveness of the GET token section by:

- Replacing the GET token image with a scalable, more realistic, 
high-quality SVG
- Replacing the side to side coin animation with a more refined rotating tilt and subtle coin reflection
- Adjusting font sizes and spacing for better visual hierarchy and readability

### Why This Matters


These visual updates make the page look more polished and professional while improving clarity across devices. The changes ensure a consistent and high-quality user experience. The enhanced animation and refined layout reinforce the sense of trust, innovation, and momentum that users should feel when interacting with $GET. 

Tested on various screen sizes. Ready for review!

skajosborn committed:
feat(get-token): update coin image size, tilt animation, and font styles

skajosborn committed:
fix(get-token): refine layout and styling for about, earn, and trust sections

skajosborn committed:
feat(get-token): made several updates to  page to improve layout, responsiveness and overall appearance

skajosborn committed:
Fix build error in  page

skajosborn committed:
chore(get-token): replaced PNGs with SVGs for better clarity and scalability

skajosborn committed:
Refactor: Made several updates and improvements to  token pg to improve overall appearance and layout including svgs in place of pngs

skajosborn committed:
Refactor: restored original icons in community section

skajosborn committed:
fixed dot on live trade badge and darkened icon backgrounds in trust section

skajosborn committed:
Enhanced tokenomics and how to buy pages and improved styling

sanaderi commented on assets/images/token/GovernancePopup.svg:
Plz remove extra images like this

sanaderi commented on nuxt.config.js:
```diff
@@ -106,7 +106,6 @@ export default defineNuxtConfig({
   // Plugins
   plugins: [
     { src: "plugins/helper.js" },
-    { src: "plugins/vue-emoji-picker.js", mode: "client" },
```
Plz don't remove this. this use in some where of project

sanaderi commented on public/images/token/GET_Tokenomics_Android_Small copy.svg:
Don't use images instead of text and styles.

sanaderi commented on public/images/token/GET_Tokenomics_Android_Small.svg:
Also remove extra images

sanaderi commented on plugins/vue-emoji-picker.js:
Don't remove this, it used somewhere in the project

sanaderi reviewed (changes requested):
This section has issue:
![image](https://github.com/user-attachments/assets/dca96a27-2707-48c6-9ca3-b7e4b5f1d917)


Font style for the card header doesn't match the design
![image](https://github.com/user-attachments/assets/e9ff258b-b9e5-435d-873d-4b936f901492)

Chart is not usable on mobile screens
![image](https://github.com/user-attachments/assets/44de8028-62a5-4451-9ecc-8ee4c03daebf)

skajosborn committed:
Update to Tokenomics and How to Buy sections of  pg. Added popups, improved layout and design.

skajosborn commented:
Pushed new changes for the Tokenomics and How to Buy sections:
	•	✅ Addressed all review requests above
	•	🎨 Updated visuals and layout refinements
	•	🧠 Added interactive popups/tooltips to the Tokenomics page
	•	✅ Build is successful 

Let me know what you think!
![Image](https://github.com/user-attachments/assets/f0dbd1bc-8e41-40a1-a1ab-4f5f4ad66292)
![Image](https://github.com/user-attachments/assets/e04e433d-be89-40ed-87da-f324bb9be24b)
![Image](https://github.com/user-attachments/assets/7c66bd6d-1e5f-4605-90c9-ad2acfc2f010)
![Image](https://github.com/user-attachments/assets/2cc0e46c-46c8-4fa0-af9b-cea87e7764a3)
![Image](https://github.com/user-attachments/assets/2296461d-4a51-4057-bdef-a97fdc26b689)

skajosborn committed:
Fixed mobile responsiveness to Tokenomics section

sanaderi commented:
Please recheck this — the chart view looks bad on mobile. We might need to redesign it for better mobile compatibility.
![image](https://github.com/user-attachments/assets/e3b93b1a-54ed-4364-a088-6e0f344ddc36)


**Also, on these cards:**
The font size of the card titles (e.g., “Set up a wallet,” “Connect to a Decentralized Exchange (DEX),” etc.) is too small — and since it's a title, it should be more prominent.

I also think it would look better if the titles were center-aligned, like the icons above them.
![image](https://github.com/user-attachments/assets/92955bde-e6f3-4ead-9e1e-90d00d1e1136)

skajosborn committed:
feat: Improve layout and responsiveness for Buy, Tokenomics, and Hero sections

skajosborn committed:
fixed casing error in TokenPurchse import

skajosborn commented:
@sanaderi I fixed the casing error for the Jupiter import. I'm not sure why they're not showing up for me in the build process. As for the Stats section, I haven't touched that at all. If it's not working or needs revision, I can look into redesigning it. Let me know what you're thinking.

sanaderi commented:
<img width="1711" height="572" alt="image" src="https://github.com/user-attachments/assets/a83cff8a-ca45-44d5-b73d-60625e44ea2d" />

sanaderi commented:
@Matinj031 Plz review and fix the remaining issues.

skajosborn commented:
> @Matinj031 Plz review and fix the remaining issues.

@Matinj031 @sanaderi  I’ve made several changes and improvements, and the page should now be about 99% complete. I’ve invested over 200 hours on this, including learning Nuxt.js from scratch, and I believe the remaining issues are likely Nuxt-specific syntax. I don’t have more time to dedicate at the moment, but if you run into any issues or have questions, feel free to reach out — I’ll be happy to help where I can.

sanaderi commented:
@skajosborn We truly appreciate you—thank you so much! We’d love to have you beside the project forever. @Matinj031, as an experienced Nuxt.js developer, will help complete this issue as soon as possible.
